app/views/matches/replay.slim

```diff
@@ -1,4 +1,5 @@
 = javascript_include_tag "page_specific/match_replay.js"
+= javascript_include_tag "page_specific/underscore-min.js"
```

Comment:
Yeah, I used it on line 92 for the filter function. You might think about incorporating it into the rest of the site as well; it's an incredibly useful library.